### PR TITLE
Fix typo in dbus property name ("OnSuccesJobMode")

### PR DESCRIPTION
**Pull request by CyberShadow**
I noticed this typo while troubleshooting something.

Not sure what the consequences of fixing it now are. Hope this helps. :)

### CODE REVIEW

**Comment by bluca** [comment]
yeah we need to add a new one with the right name, and mark the old one as deprecated in the comments but leave it there, I'm afraid

**Review by bluca — APPROVED** [review]
lgtm

File: src/core/dbus-unit.c [hunk 1]
```diff
@@ -905,7 +905,8 @@ const sd_bus_vtable bus_unit_vtable[] = {
         SD_BUS_PROPERTY("RefuseManualStop", "b", bus_property_get_bool, offsetof(Unit, refuse_manual_stop), SD_BUS_VTABLE_PROPERTY_CONST),
         SD_BUS_PROPERTY("AllowIsolate", "b", bus_property_get_bool, offsetof(Unit, allow_isolate), SD_BUS_VTABLE_PROPERTY_CONST),
         SD_BUS_PROPERTY("DefaultDependencies", "b", bus_property_get_bool, offsetof(Unit, default_dependencies), SD_BUS_VTABLE_PROPERTY_CONST),
-        SD_BUS_PROPERTY("OnSuccesJobMode", "s", property_get_job_mode, offsetof(Unit, on_success_job_mode), SD_BUS_VTABLE_PROPERTY_CONST),
+        SD_BUS_PROPERTY("OnSuccesJobMode", "s", property_get_job_mode, offsetof(Unit, on_success_job_mode), SD_BUS_VTABLE_PROPERTY_CONST), /* deprecated */
```
Comment (yuwata) [inline src/core/dbus-unit.c]: Please add `SD_BUS_VTABLE_HIDDEN` flag.

**Review by yuwata — CHANGES_REQUESTED** [review]

**Review by yuwata — APPROVED** [review]
LGTM.

**Comment by poettering** [comment]
lgtm

**Pull request merged**